R/4_standard_error.R

```diff
@@ -1,33 +1,34 @@
 #' Standard Errors
 #'
 #' `standard_error()` attempts to return standard errors of model
-#' parameters, while `standard_error_robust()` attempts to return robust
-#' standard errors.
+#' parameters
 #'
 #' @param model A model.
 #' @param force Logical, if `TRUE`, factors are converted to numerical
```

Comment:
Not related to your changes, but I don't understand what this argument is supposed to do, and it's not used anywhere. Should we remove it?

Reply:
https://github.com/easystats/parameters/blob/f957b8f71444ca24da0762ae583ad8c0d5e75fce/R/methods_base.R

Reply:
Oh, okay. I didn't realize that these functions could be applied to vectors to compute the standard error of the mean and not only models. 

We should clarify that for the `model` and `force` arguments. 

```
model A model. Can also be a vector or data frame to compute the standard error of the mean for one or more variables. 
force When `model` is a vector or data frame, if `force = TRUE`, ...
```

Reply:
Ooof. Is this `force` argument ever a good idea? Feels like users should be responsible for doing dumb stuff on their own... (Also, the word is pretty generic; could lead to confusion.)

Reply:
It's kind of "historical reason", we use the `force` argument in different places (related to factors), e.g. in `standardize()` to force standardization of factors as well.

>  Feels like users should be responsible for doing dumb stuff

With this attitude, we'll never come beyond "p < 0.05 means evidence of an effect" ;-) We have an educational function / responsibility as well.

Reply:
![toni](https://user-images.githubusercontent.com/26301769/154574998-c5e893db-840a-411b-85a4-0f7b6751450b.gif)

(this smiley was named after me, because every now and then I appear as wisenheimer :-))